crates/iota-protocol-config/src/lib.rs

```diff
@@ -1983,6 +1990,27 @@ impl ProtocolConfig {
                         cfg.feature_flags.congestion_control_min_free_execution_slot = true;
                     }
                 }
+                9 => {
+                    // Enable round prober in consensus.
+                    cfg.feature_flags.consensus_round_prober = true;
+                    // Enable distributed vote scoring.
+                    cfg.feature_flags
+                        .consensus_distributed_vote_scoring_strategy = true;
+                    cfg.feature_flags.consensus_linearize_subdag_v2 = true;
+                    // Enable smart ancestor selection for testnet
+                    cfg.feature_flags.consensus_smart_ancestor_selection = true;
+                    // Enable probing for accepted rounds in round prober for testnet
+                    cfg.feature_flags
+                        .consensus_round_prober_probe_accepted_rounds = true;
+                    // Enable zstd compression for consensus in testnet
+                    cfg.feature_flags.consensus_zstd_compression = true;
```

Comment:
Shouldn't these be in an `if chain == Chain::Testnet {}` block if they are just for the testnet?

Reply:
arg, copy-paste, will fix the comments